### DISCUSSION TITLE
studio local dev setup keeps redirecting to something went wrong

### DISCUSSION THREAD

**Discussion by aquibbaig** [supabase/supabase#14179] [General]
I don't know if this is an issue with local setup for everyone but I've tried about 3 ways of launching a supabase dev-setup and at this point, I'm exhausted. In every case, when I launch the studio on :8082 it keeps redirecting me to the "Something went wrong" page. Seems like either the session is null or the `isRedirecting` flag is false.

Here are some of my approaches:
## Case 1: Turbo startup
- In the root folder, do an npm install & npm run dev as specified in https://github.com/supabase/supabase/blob/master/DEVELOPERS.md

## Case 2: Self hosted setup
- Followed this guide to create a custom supabase project locally. https://supabase.com/docs/guides/getting-started/local-development
- The idea was to launch this project using `supabase start`, use the creds in the env file of the studio & run just the studio using `npm run dev` but it still produces the same issue.

## Case 3: Running everything using docker-compose.dev.yml
- Followed this guide https://github.com/supabase/supabase/tree/master/studio. 
- Running a docker compose on `docker-compose.dev.yml` spins off respective containers for everything & I can launch the studio on the specified URL. 
- However I do not understand is whenever I make a code change do I need to relaunch everything to see the change here or how does this work? If yes, this brings down the dev experience by a ton.

Any help would be truly appreciated here. Thanks and good luck! :+1:

**Comment by Hallidayo** [comment]
Hi @aquibbaig do you get any errors in your browser console?

**Comment by aquibbaig** [comment]
Nope. nothing @Hallidayo 
![Attachment](https://github.com/supabase/supabase/assets/26324376/6103cd75-cbb8-4589-b0a7-1505774cb650)
